### DISCUSSION TITLE
NPCs cannot access damaged walls surrounding the docking hatch

### DISCUSSION THREAD

**Discussion by coldninja9** [FakeFishGames/Barotrauma#15261] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

On the "Herja" submarine an NPC glitch occurs when the hull surrounding the docking hatch is damaged where NPC's will attempt to fix the hull but cannot access it. 

### Reproduction steps

1. Select "Herja" submarine
2. Assign any NPC to "fix leaks" 
3. Damage hull surrounding the docking port towards the top of the submarine
4. Observe

### Bug prevalence

Happens regularly

### Single player or multiplayer?

Single player

### Version

v1.6.19.1 (Unto the Breach Update Hotfix 2)

### Which operating system did you encounter this bug on?

Windows

**Comment by BOTxMALWARY** [comment]
Yeah it is so annoying

**Comment by Regalis11** [comment]
Thank you for the report! This has been fixed in our private repo now, fix coming up in the next unstable build and the upcoming release.
